.github/workflows/pytest.yml

```diff
@@ -24,13 +24,7 @@ jobs:
         with:
           python-version: "3.10"
           cache: "pip"
-
-      - name: Cache python environment
-        id: cache-python-env
-        uses: actions/cache@v3
-        with:
-          path: ${{ env.pythonLocation }}
-          key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
```

Comment:
How are key managed by `cache-dependency-path`? Are we sure this is not being shared elsewhere?

Reply:
The cache key is managed the same way to the previous implementation, see https://github.com/actions/setup-python/blob/65d7f2d534ac1bc67fcd62888c5f4f3d2cb2b236/docs/adrs/0000-caching-dependencies.md#decision